x-pack/plugins/siem/public/apps/start_app.tsx

```diff
@@ -33,21 +34,24 @@ export const startApp = async (libs: AppFrontendLibs) => {
   libs.framework.render(
     <EuiErrorBoundary>
       <I18nContext>
-        <ReduxStoreProvider store={store}>
-          <ApolloProvider client={libs.apolloClient}>
-            <ThemeProvider
-              theme={() => ({
-                eui: libs.framework.darkMode ? euiDarkVars : euiLightVars,
-                darkMode: libs.framework.darkMode,
-              })}
-            >
-              <KibanaConfigContext.Provider value={libs.framework}>
-                <PageRouter history={history} />
-              </KibanaConfigContext.Provider>
-            </ThemeProvider>
-            <ErrorToast />
-          </ApolloProvider>
-        </ReduxStoreProvider>
+        <ManageGlobalToaster>
+          <ReduxStoreProvider store={store}>
+            <ApolloProvider client={libs.apolloClient}>
+              <ThemeProvider
+                theme={() => ({
+                  eui: libs.framework.darkMode ? euiDarkVars : euiLightVars,
+                  darkMode: libs.framework.darkMode,
+                })}
+              >
+                <KibanaConfigContext.Provider value={libs.framework}>
+                  <PageRouter history={history} />
+                </KibanaConfigContext.Provider>
+              </ThemeProvider>
+              <ErrorToast />
```

Comment:
nit: I wonder if we should rename this to remove any confusion around there being multiple global toast lists. Perhaps `ErrorToastDispatcher` or something along those lines?